integrated_channels/api/v1/logs/views.py

```diff
@@ -24,32 +25,83 @@
     LearnerDataTransmissionAudit,
 )
 
-from .serializers import ContentSyncStatusSerializer, GenericLearnerSyncStatusSerializer, LearnerSyncStatusSerializer
+from .serializers import ContentSyncStatusSerializer, LearnerSyncStatusSerializer
 
 LOGGER = logging.getLogger(__name__)
 
 
+class ReportingSyncStatusPagination(PageNumberPagination):
+    page_size = 10
+
+    def get_paginated_response(self, data):
+        return Response(OrderedDict([
+            ('pages_count', self.page.paginator.num_pages),
+            ('count', self.page.paginator.count),
+            ('next', self.get_next_link()),
+            ('previous', self.get_previous_link()),
+            ('results', data)
+        ]))
+
+
+class InvalidSortByParamException(exceptions.APIException):
```

Comment:
nit: These exceptions all have the same fields, maybe have a base class for them?